conf/elb/load-balancers/example-load-balancer.json

```diff
@@ -0,0 +1,52 @@
+{
+  "listeners": {
+    "includes": {
+       "example-listener": {
```

Comment:
In other places, we do

```
{
  template: "example-listener",
  vars: {
    "ssl-cert": "value"
  }
}
```
It seems a little more obvious what's going on there.